docs/modules/ROOT/pages/how-to-guides/configuring-builds/proc_customize_build_pipeline.adoc

```diff
@@ -0,0 +1,99 @@
+= Customizing the build pipeline
+
+You can customize the way that {ProductName} builds components, to meet your specific needs, in two ways:
+
+* By changing the parameters of the build pipeline of a component
```

Comment:
Make it action focused:

You can customize in two ways:

1. Change the parameters of the build pipeline of a component. 
2. Extend the pipeline with your own tasks or delete existing tasks. 

Note, we tend to avoid parenthesis unless it's giving an acronym.